src/server/test/capacity_unit_calculator_test.cpp

```diff
@@ -50,11 +51,13 @@ class capacity_unit_calculator_test : public pegasus_server_test_base
     std::unique_ptr<mock_capacity_unit_calculator> _cal;
 
 public:
-    const dsn::blob key = dsn::blob::create_from_bytes("key");
+    dsn::blob key, hash_key;
 
     capacity_unit_calculator_test() : pegasus_server_test_base()
     {
         _cal = dsn::make_unique<mock_capacity_unit_calculator>(_server.get());
+        pegasus_generate_key(key, dsn::blob::create_from_bytes("h"), dsn::blob());
+        hash_key = dsn::blob::create_from_bytes("key");
```

Comment:
`hash_key` is duplicate with `hash_key` in check_and_set/check_and_mutate tests.

Reply:
Yes, but I think it is ok to override it in check_and_set/check_and_mutate tests.

Reply:
I think it is better to rename `hash_key` to `test_hash_key` or `HASH_KEY`, or rename `hash_key` in check_and_set/check_and_mutate tests, to avoid unnecessary misunderstanding.